source/2019-05-25-the-ember-times-issue-100.md

```diff
@@ -93,13 +94,13 @@ Please thank [@Alonski](https://github.com/Alonski), [@maxwondercorn](https://gi
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [Codemod for ES6 Class Syntax Including Decorators🤖](https://github.com/ember-codemods/ember-es6-class-codemod)
+Excited about the chance to try out the ✨**native decorator support**✨ announced as part of the [release of Ember.js 3.10](https://blog.emberjs.com/2019/05/21/ember-3-10-released.html)? The good news is there's a [codemod](https://github.com/ember-codemods/ember-es6-class-codemod) to help you out with that!
+
+Note that this part of the codemod is a configurable option in the transforms. If set to true, it will transform an object's properties to decorators wherever required. Also note that, unless disabled, this codemod will by default handle transforming object properties to class fields.
```

Comment:
It seems there's an issue with the term `disabled` when running the alex-linter. What do you think about

```suggestion
Note that this part of the codemod is a configurable option in the transforms. If set to true, it will transform an object's properties to decorators wherever required. Also note that, unless turned off, this codemod will by default handle transforming object properties to class fields.
```